### PR TITLE
Bug 1623626 - Add crash stacks to crash schema

### PR DESCRIPTION
**Pull request by wlach**
Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by wlach** [comment]
Still a work in progress, per error rate above. Plz hold off on review for now

**Comment by wlach** [comment]
Ok, ready for review now. In addition to the the integration tests, I also validated this works ok with all the crash stacks submitted with crash stacks on March 1st.

@jklukas -- could you validate that these schema additions should be interpreted ok by our bigquery machinery? it's rather larger and more complex than usual.

File: templates/telemetry/crash/crash.4.schema.json [hunk 1]
```diff
@@ -162,6 +162,72 @@
         "processType": {
           "type": "string"
         },
+        "stackTraces": {
+          "type": ["object", "null"],
+          "properties": {
+            "crash_info": {
+              "type": "object",
+              "properties": {
+                "address": @TELEMETRY_HEXADDRESS_1_JSON@,
```
Comment (jklukas) [inline templates/telemetry/crash/crash.4.schema.json]: Do we really want to throw out the ping if one of these hex fields isn't in the expected format? Should we just let these be strings without validation?

Comment (wlach) [reply templates/telemetry/crash/crash.4.schema.json]: On the fence about this -- the fact that all the pings we're seeing in the field are formatting this correctly make me think we should be strict about this so we notice if people start submitting garbage. The main point of this telemetry is to allow us to create a signature of crashes that are being submitted, I'd like to be notified directly if that seems to be impossible.

**Comment by wlach** [comment]
Good idea re: the test cases, I'll make sure to add some for the permutations I found.

**Pull request merged**